apps/condo/domains/common/hooks/useValidations.ts

```diff
@@ -41,6 +42,7 @@ export const useValidations: UseValidations = (settings = {}) => {
     const FieldIsTooLongMessage = intl.formatMessage({ id: 'ValueIsTooLong' })
     const NumberIsNotValidMessage = intl.formatMessage({ id: 'NumberIsNotValid' })
     const TinValueIsInvalidMessage = intl.formatMessage({ id: 'pages.organizations.tin.InvalidValue' })
+    const EmailsAreInvalidMessage = intl.formatMessage({ id: 'condorb.emails.invalidMessage' })
```

Comment:
you should change `id`, this translation can be used in any domain. 
We have `ADR15` about translations in `Confluence`